src/wyzeapy/utils.py

```diff
@@ -72,37 +72,45 @@ def create_password(password: str) -> str:
     return hashlib.md5(hex2.encode()).hexdigest()
 
 
-def check_for_errors_standard(call, response_json: Dict[str, Any]) -> None:
+def check_for_errors_standard(service, response_json: Dict[str, Any]) -> None:
     if response_json['code'] != ResponseCodes.SUCCESS.value:
         if response_json['code'] == ResponseCodes.PARAMETER_ERROR.value:
             raise ParameterError(response_json)
         elif response_json['code'] == ResponseCodes.ACCESS_TOKEN_ERROR.value:
-            call._auth_lib.token.expired = True
+            service._auth_lib.token.expired = True
             raise AccessTokenError("Access Token expired, attempting to refresh")
         elif response_json['code'] == ResponseCodes.DEVICE_OFFLINE.value:
             return
         else:
             raise UnknownApiError(response_json)
 
 
-def check_for_errors_lock(response_json: Dict[str, Any]) -> None:
+def check_for_errors_lock(service, response_json: Dict[str, Any]) -> None:
     if response_json['ErrNo'] != 0:
         if response_json.get('code') == ResponseCodes.PARAMETER_ERROR.value:
             raise ParameterError
         elif response_json.get('code') == ResponseCodes.ACCESS_TOKEN_ERROR.value:
-            raise AccessTokenError
+            service._auth_lib.token.expired = True
+            raise AccessTokenError("Access Token expired, attempting to refresh")
         else:
             raise UnknownApiError(response_json)
 
 
-def check_for_errors_iot(response_json: Dict[Any, Any]) -> None:
+def check_for_errors_iot(service, response_json: Dict[Any, Any]) -> None:
     if response_json['code'] != 1:
-        raise UnknownApiError(response_json)
-
+        if response_json.get('code') == ResponseCodes.ACCESS_TOKEN_ERROR.value:
+            service._auth_lib.token.expired = True
+            raise AccessTokenError("Access Token expired, attempting to refresh")
+        else:
+            raise UnknownApiError(response_json)
 
-def check_for_errors_hms(response_json: Dict[Any, Any]) -> None:
+def check_for_errors_hms(service, response_json: Dict[Any, Any]) -> None:
     if response_json['message'] is None:
-        raise AccessTokenError
+        if response_json.get('code') == ResponseCodes.ACCESS_TOKEN_ERROR.value:
```

Comment:
I'm not sure what is returned for the hms service, so I'm making the assumption its similar to the others...

Reply:
Actually I see it is called once at startup. Do you know why we test for the message (or lack of) here vs the response code like the rest?

Reply:
I honestly don't. I don't have the HMS from Wyze, so I never really looked much at that code or response. Looking at that code though, it looks like the response from Wyze is probably inconsistent with the rest of the responses (which also isn't surprising).

Reply:
I faked an access token error and this is the response, so it looks like testing for lack of a message key is not correct, and that it does throw the access token error, but I don't have HMS either, this is called once at startup by `_get_plan_binding_list_by_user()`

`{'code': 2001, 'current': 0, 'hash': '1', 'instance_id': 'a9bfe493-5a58-4546-a05c-e8ee9915a815', 'message': 'Access token error', 'total': 0, 'version': 1}`

Reply:
Well, that's interesting. Yeah, the 2001 code is the same code that the other checks are using to check against. So I guess that this should realistically be checking the same way as everything else is, to be honest.

Reply:
Of course the HMS service returns 2001 as an int while everything else is '2001'...typical

Reply:
Yeah, that's kinda how Wyze rolls. Just cast it to a string. Python's pretty good about not doing wacky things during the conversion.
`if str(response_json.get('code')) == ResponseCodes.ACCESS_TOKEN_ERROR.value:`